### ISSUE TITLE
grpc-benchmarks and grpc-interop-testing in google-cloud-bom

### ISSUE THREAD

**Issue by suztomo** [GoogleCloudPlatform/cloud-opensource-java#1004]
Jeff's [import_boms branch](https://github.com/googleapis/google-cloud-java/pull/6554) is trying to add all gRPC artifacts into google-cloud-bom.

Among these gRPC artifacts, grpc-benchmarks and grpc-interop-testing have linkage errors in their transitive dependencies.

```
mockito-core-2.25.1.jar is at:
  io.grpc:grpc-benchmarks:1.24.1 (compile) / org.mockito:mockito-core:2.25.1 (compile)
byte-buddy-1.9.7.jar is at:
  io.grpc:grpc-benchmarks:1.24.1 (compile) / org.mockito:mockito-core:2.25.1 (compile) / net.bytebuddy:byte-buddy:1.9.7 (compile)
gwt-user-2.8.2.jar is at:
  io.grpc:grpc-interop-testing:1.24.1 (compile) / com.google.truth:truth:0.45 (compile) / com.google.gwt:gwt-user:2.8.2 (provided)
```

https://fusion.corp.google.com/runanalysis/buildlogs/prod%3Acloud-devrel%2Fclient-libraries%2Fjava%2Fgoogle-cloud-java%2Fpresubmit%2Flinkage-monitor/prod%3Acloud-devrel%2Fclient-libraries%2Fjava%2Fgoogle-cloud-java%2Fpresubmit%2Flinkage-monitor/KOKORO/b80fa904-056f-409c-99fe-c01a3e3f2776/0/prod%3Acloud-devrel%2Fclient-libraries%2Fjava%2Fgoogle-cloud-java%2Fpresubmit%2Flinkage-monitor/Targets

**Comment by suztomo** [comment]
@chingor13 Do you see the following statement correct (or not)?

The import_boms branch is trying to add all gRPC artifacts (via grpc-bom) into google-cloud-bom's dependencyManagement section.

**Comment by elharo** [comment]
I disagree that it makes sense to skip these two artifacts. If they're not supposed to be used by users, google-cloud-java shouldn't import them and grpc-bom shouldn't publish them.

**Comment by elharo** [comment]
I don't immediately see where that branch is pulling in grpc-bom. Can you give me a line number?

**Comment by suztomo** [comment]
It turned out that it was unexpected import of grpc-bom.
